3_javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,69 @@
+//Flight booking
+function getFullname(firstname, surname, useFormalName, gender) {
+  if (useFormalName == true && gender == "man") {
+    return `Lord ${firstname} ${surname}`;
+  } else if (useFormalName == true && gender == "woman") {
+    return `Miss ${firstname} ${surname}`;
+  } else {
+    return `${firstname} ${surname}`;
+  }
+}
+
+let fullname1 = getFullname("Sevda", "Naziri", true, "woman");
+let fullname2 = getFullname("John", "Wick", true, "man");
+
+console.log(fullname1);
+console.log(fullname2);
+
+//Event application
+let daysUntilEvent = 8;
+
+function getEventWeekday(days) {
+  var date = new Date();
+  date.setDate(date.getDate() + days);
+  return date;
+}
+console.log(getEventWeekday(daysUntilEvent));
+
```

Comment:
🌟 Great effort on tackling this challenge!
🤖 Your code looks good, and you've used the `Date` object effectively to calculate the date of the event. Nice work!
💡What you are getting returned from your function is the date of the event, what the question is asking is the day of the event like Thursday, Monday and ...
🗓️ You're on the right track with your use of `date.getDate()`, but you can take it a step further by converting the date to a weekday. Investigate the `getDay()` method.